docs/zh_cn/reference/how_to_set_up_metadata_engine.md

```diff
@@ -384,6 +384,56 @@ juicefs mount -d "mysql://user@(192.168.1.6:3306)/juicefs?tls=true" /mnt/jfs
 
 更多 MariaDB 数据库的地址格式示例，[点此查看](https://github.com/Go-SQL-Driver/MySQL/#examples)。
 
+## OceanBase
+
+[OceanBase](https://www.oceanbase.com/)是一款自主研发的分布式关系型数据库，专为处理海量数据和高并发事务而设计，具备高性能、强一致性和高可用性的特点。其独特的多副本架构和弹性扩展能力能够满足金融级可靠性需求，同时兼容主流数据库协议，支持复杂查询与事务处理，适用于金融、电信、互联网等关键业务场景。OceanBase在保证数据一致性的同时，提供优异的扩展性和成本效益，帮助企业实现从单机到分布式的无缝过渡。
+
+OceanBase 与 MySQL 高度兼容，在使用上没有任何差别，同样需要提前创建数据库，创建和挂载文件系统时，保持与 MySQL 相同的语法。
```

Comment:
These are duplicated, we may remove those for MariaDB also.

Reply:
Already deleted the duplicate content in the article how_to_set_up_metadata_engine.md. Please check if it is OK.